### PR TITLE
Opower integration support for multiple meters on single account

### PR DESCRIPTION
**Pull request by c0mputerguru**
## Proposed change

Documentation updates for the fix to https://github.com/home-assistant/core/issues/108260

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108616
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #108260

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by tronikos — APPROVED** [review]

File: source/_integrations/opower.markdown [hunk 1]
```diff
@@ -91,6 +91,10 @@ For gas:
 
 Note the unit for gas is CCF (centum cubic feet). 1 CCF is one hundred cubic feet which is equivalent to 1 therm.
 
+Depending on the utility and your setup, there may be multiple meters for a single utility account.
+If this is the case, your sensors will use a UUID to distinguish the data streams within that single account.
```
Comment (frenck) [inline source/_integrations/opower.markdown]: UUID is a rather technical term, can we avoid that?

Comment (c0mputerguru) [reply source/_integrations/opower.markdown]: Removed it and replaced to explain that the id will not be tied back to anything the energy provider provides.

File: source/_integrations/opower.markdown [hunk 2]
```diff
@@ -91,6 +91,9 @@ For gas:
 
 Note the unit for gas is CCF (centum cubic feet). 1 CCF is one hundred cubic feet which is equivalent to 1 therm.
 
+Depending on the utility and your setup, there may be multiple meters for a single utility account.
+If this is the case, your sensors will use a unique id that does not tie back to any identifier used by the utility to distinguish the data streams within that single account.
```
Comment (frenck) [inline source/_integrations/opower.markdown]:
```suggestion
Depending on the utility and your setup, multiple meters may exist for a single utility account.
If this is the case, your sensors will use a unique ID that does not tie back to any identifier used by the utility to distinguish the data streams within that single account.
```

**Review by frenck — APPROVED** [review]
Thanks, @c0mputerguru 👍 

../Frenck

**Comment by tronikos** [comment]
Since the code was reverted in https://github.com/home-assistant/core/pull/117476, this needs to be reverted too.

**Pull request merged**